roles/network_plugin/calico/tasks/install.yml

```diff
@@ -154,6 +154,17 @@
     - inventory_hostname == groups['kube-master'][0]
     - 'calico_conf.stdout == "0"'
 
+- name: Set Service External IPs to empty list
```

Comment:
This section can be deleted or changed the task below:
```
 - name: Populate Service External IPs 
  set_fact:
    _service_external_ips: "{{ _service_external_ips|default([]) + [ {'cidr': item} ] }}" 
```
to
```
 - name: Populate Service External IPs 
  set_fact:
    _service_external_ips: "{{ _service_external_ips + [ {'cidr': item} ] }}" 
```

Reply:
It is nice to remove this section.

Reply:
removed